tools/update_extern.py

```diff
@@ -0,0 +1,119 @@
+"""
+Updates all the libraries in ``sunpy/extern``
+"""
+
+import os
+import shutil
+from zipfile import ZipFile
+import requests
```

Comment:
Can we not just use Parfive (the sunpy downloader instead of having to use requests and tqdm?

Reply:
You can see how to use it here: https://github.com/Cadair/parfive

Reply:
Yeah, `requests` is a third party packages that SunPy does not directly depend on, so ideally it would be better to use `parfive` to download the zip file and [`urllib.request`](https://docs.python.org/3/library/urllib.request.html#module-urllib.request) from the Python Standard Library for getting the HTTP response to parse as JSON.